### ISSUE TITLE
Multiple times linkedRepository

### ISSUE THREAD

**Issue by dakue** [mibexsoftware/bamboo-plan-dsl-plugin#1]
Hello,

I'm using a DSL which looks nearly like:

```
project('MYPROJECT') {
    name 'My project'

    plan('MYPLAN') {
        name 'My plan'

       scm {
          linkedRepository('repository-one')
          linkedRepository('repository-two')
       }

        stage('My stage') {
            description 'My stage'
            manual false

            job('BUILD') {
                name 'Maven build job'

                tasks {
                   checkout('checkout repository-one') {
                        repository('repository-one') {}
                    }

                    checkout('repository-two') {
                        repository('repository-two') {
                            checkoutDirectory 'two'
                        }
                    }
                }
            }
        }
    }
}
```

When the 'Seed' task uses this plan I get the error message:

```
failed due to an error in the provided DSL script(s). Details: (TransactionTemplate.java:133): could not find a repository for repository-two
```

Kind regards,

Daniel

**Comment by mrueegg** [comment]
Hi Daniel,

Thanks for your bug report. I was able to fix the bug. I will release a new plug-in version today and reply here once it is out.

**Comment by mrueegg** [comment]
Fixed with release 1.2.1. Thanks for reporting this bug!
